python/utilities/MixingTimeResolution_Aux.cpp

```diff
@@ -0,0 +1,14 @@
+
+#include <pybind11/pybind11.h>
+
+#include <goofit/PDFs/physics/MixingTimeResolution_Aux.h>
+
+using namespace GooFit;
+namespace py = pybind11;
+
```

Comment:
Virtual functions needed to be binded as well, and that makes sense. I am just confused that I am getting successful tests and build on my computer but the pytest is failing over here